### PR TITLE
Resolves #1796

### LINKED ISSUE #1796: It is not practical
Missing examples of use, combinations with other instructions and often do not show the unit of the result of a command.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3e73e02c-6320-8ebc-5bea-5cf288340ede
* Version Independent ID: 201b54ab-7995-f51c-df21-34c0b48de098
* Content: [New-Partition (storage)](https://docs.microsoft.com/en-us/powershell/module/storage/new-partition?view=win10-ps)
* Content Source: [docset/windows/storage/New-Partition.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/live/docset/windows/storage/New-Partition.md)
* Product: **w10**
* Technology: **powershell-windows**
* GitHub Login: @andreabarr
* Microsoft Alias: **v-anbarr**

### PR DESCRIPTION
**Pull request by velkovb**
Provide more examples
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1796

### CODE REVIEW

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by dariomws — APPROVED** [review]

File: docset/windows/storage/New-Partition.md [hunk 1]
```diff
@@ -60,31 +60,33 @@ Note: This cmdlet does not support creating dynamic volumes.
...
+PS C:\> Get-Disk | Where-Object PartitionStyle -Eq "RAW" | Initialize-Disk -PassThru | New-Partition -AssignDriveLetter -UseMaximumSize | Format-Volume
 ```
-This line gets all disk objects and then pipes the objects to the next command.
-PS C:\>Get-Disk |
 
-This line selects only objects where the PartitionStyle property value equals "RAW", and then pipes the objects to the next command.
-PS C:\>Where-Object PartitionStyle -Eq "RAW" |
+This example uses five cmdlets and the pipeline to get all disks, filter them for only RAW, unpartitioned disks, initialize the disks, partition the disks, and then format them.
+
+### Example 3: Create a new EFI partition on on GTP disk 2
```
Comment (get-itips) [inline docset/windows/storage/New-Partition.md:76]:
```suggestion
### Example 3: Create a new EFI partition on GTP disk 2
```

**Review by get-itips — COMMENTED** [review]
Please remove the extra "on" on the other articles. Thank you

**Review by tseward — APPROVED** [review]
Approve pending requested changes

File: docset/windows/storage/New-Partition.md [hunk 2]
```diff
@@ -60,31 +60,33 @@ Note: This cmdlet does not support creating dynamic volumes.
...
 
 ### Example 1: Create a new partition on disk 1
-```
-PS C:\> New-Partition -DiskNumber 1 -UseMaximumSize -AssignDriveLetter
+```powershell
+PS C:\> New-Partition -DiskNumber 1 -UseMaximumSize -DriveLetter T
 ```
 
-This example creates a new partition on disk 1, using the maximum available space, and automatically assigning a drive letter.
+This example creates a new partition on disk 1, using the maximum available space, and assign a drive letter T.
```
Comment (JohanFreelancer9) [inline docset/windows/storage/New-Partition.md:67]:
```suggestion
This example creates a new partition on disk 1 using the maximum available space, and assigns a drive letter T.
```

File: docset/windows/storage/New-Partition.md [hunk 3]
```diff
@@ -60,31 +60,33 @@ Note: This cmdlet does not support creating dynamic volumes.
...
 ### Example 2: Get all RAW disks, initialize the disks, partition, and format them
+```powershell
+PS C:\> Get-Disk | Where-Object PartitionStyle -Eq "RAW" | Initialize-Disk -PassThru | New-Partition -AssignDriveLetter -UseMaximumSize | Format-Volume
 ```
-This line gets all disk objects and then pipes the objects to the next command.
-PS C:\>Get-Disk |
 
-This line selects only objects where the PartitionStyle property value equals "RAW", and then pipes the objects to the next command.
-PS C:\>Where-Object PartitionStyle -Eq "RAW" |
+This example uses five cmdlets and the pipeline to get all disks, filter them for only RAW, unpartitioned disks, initialize the disks, partition the disks, and then format them.
```
Comment (JohanFreelancer9) [inline docset/windows/storage/New-Partition.md:74]:
```suggestion
This example uses five cmdlets and the pipeline to get all disks, filters them for only RAW, unpartitioned disks, initializes the disks, partitions the disks, and then formats them.
```

File: docset/winserver2012-ps/storage/New-Partition.md [hunk 4]
```diff
@@ -51,31 +51,33 @@ Note: This cmdlet does not support creating dynamic volumes.
...
 
 ### Example 1: Create a new partition on disk 1
-```
-PS C:\> New-Partiton -DiskNumber 1 -UseMaximumSize -AssignDriveLetter
+```powershell
+PS C:\> New-Partition -DiskNumber 1 -UseMaximumSize -DriveLetter T
 ```
 
-This example creates a new partition on disk 1, using the maximum available space, and automatically assigning a drive letter.
+This example creates a new partition on disk 1, using the maximum available space, and assign a drive letter T.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/storage/New-Partition.md:58]:
```suggestion
This example creates a new partition on disk 1 using the maximum available space and assigns a drive letter T.
```

File: docset/windows/storage/New-Partition.md (hunk 3 above)
Comment (JohanFreelancer9) [inline docset/windows/storage/New-Partition.md:74]:
```suggestion
This example uses five cmdlets and the pipeline to get all disks, filter them for only RAW, unpartitioned disks, initialize the disks, partition the disks, and then to format them.
```

File: docset/winserver2012-ps/storage/New-Partition.md [hunk 5]
```diff
@@ -51,31 +51,33 @@ Note: This cmdlet does not support creating dynamic volumes.
...
 ### Example 2: Get all RAW disks, initialize the disks, partition, and format them
+```powershell
+PS C:\> Get-Disk | Where-Object PartitionStyle -Eq "RAW" | Initialize-Disk -PassThru | New-Partition -AssignDriveLetter -UseMaximumSize | Format-Volume
 ```
-This line gets all disk objects and then pipes the objects to the next command.
-PS C:\>Get-Disk |
 
-This line selects only objects where the PartitionStyle property value equals "RAW", and then pipes the objects to the next command.
-PS C:\>Where-Object PartitionStyle -Eq "RAW" |
+This example uses five cmdlets and the pipeline to get all disks, filter them for only RAW, unpartitioned disks, initialize the disks, partition the disks, and then format them.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/storage/New-Partition.md:65]:
```suggestion
This example uses five cmdlets and the pipeline to get all disks, filter them for only RAW, unpartitioned disks, initialize the disks, partition the disks, and then to format them.
```

File: docset/winserver2012-ps/storage/New-Partition.md [hunk 6]
```diff
@@ -51,31 +51,33 @@ Note: This cmdlet does not support creating dynamic volumes.
...
+
+### Example 3: Create a new EFI partition on on GTP disk 2
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/storage/New-Partition.md:67]:
```suggestion
### Example 3: Create a new EFI partition on GTP disk 2
```

File: docset/winserver2012-ps/storage/New-Partition.md [hunk 7]
```diff
@@ -51,31 +51,33 @@ Note: This cmdlet does not support creating dynamic volumes.
...
+```powershell
+PS C:\> New-Partition -DiskNumber 2 -Size 500MB -GptType "{c12a7328-f81f-11d2-ba4b-00a0c93ec93b}"
+```
 
-This line initializes all Disk objects in the pipeline and then pipes the objects to the next cmdlet.
-PS C:\>Initialize-Disk -PassThru |
+This example creates a new EFI partition on disk 2 with size 500MB.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/storage/New-Partition.md:72]:
```suggestion
This example creates a new EFI partition on disk 2 with size 500 MB.
```

File: docset/winserver2012-ps/storage/New-Partition.md [hunk 8]
```diff
@@ -51,31 +51,33 @@ Note: This cmdlet does not support creating dynamic volumes.
...
 
-This line formats all newly partitioned disks.
-PS C:\>Format-Volume
+### Example 4: Create a Windows/system partition on MBR disk 0
+```powershell
+PS C:\> New-Partition -DiskNumber 0 -Size 100GB -MbrType IFS -IsActive
 ```
 
-This example uses five cmdlets and the pipeline to get all disks, filter them for only RAW, unpartitioned disks, initialize the disks, partition the disks, and then format them.
+This example creates a new Windows/system partition on MBR disk 0 with size 100GB.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/storage/New-Partition.md:80]:
```suggestion
This example creates a new Windows/system partition on MBR disk 0 with size 100 GB.
```

File: docset/winserver2012r2-ps/storage/New-Partition.md [hunk 9]
```diff
@@ -59,31 +59,33 @@ Note: This cmdlet does not support creating dynamic volumes.
...
 
 ### Example 1: Create a new partition on disk 1
-```
-PS C:\> New-Partition -DiskNumber 1 -UseMaximumSize -AssignDriveLetter
+```powershell
+PS C:\> New-Partition -DiskNumber 1 -UseMaximumSize -DriveLetter T
 ```
 
-This example creates a new partition on disk 1, using the maximum available space, and automatically assigning a drive letter.
+This example creates a new partition on disk 1, using the maximum available space, and assign a drive letter T.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/storage/New-Partition.md:66]:
```suggestion
This example creates a new partition on disk 1 using the maximum available space and assigns a drive letter T.
```

File: docset/winserver2012r2-ps/storage/New-Partition.md [hunk 10]
```diff
@@ -59,31 +59,33 @@ Note: This cmdlet does not support creating dynamic volumes.
...
 ### Example 2: Get all RAW disks, initialize the disks, partition, and format them
+```powershell
+PS C:\> Get-Disk | Where-Object PartitionStyle -Eq "RAW" | Initialize-Disk -PassThru | New-Partition -AssignDriveLetter -UseMaximumSize | Format-Volume
 ```
-This line gets all disk objects and then pipes the objects to the next command.
-PS C:\>Get-Disk |
 
-This line selects only objects where the PartitionStyle property value equals "RAW", and then pipes the objects to the next command.
-PS C:\>Where-Object PartitionStyle -Eq "RAW" |
+This example uses five cmdlets and the pipeline to get all disks, filter them for only RAW, unpartitioned disks, initialize the disks, partition the disks, and then format them.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/storage/New-Partition.md:73]:
```suggestion
This example uses five cmdlets and the pipeline to get all disks, filter them for only RAW, unpartitioned disks, initialize the disks, partition the disks, and then to format them.
```

File: docset/windows/storage/New-Partition.md [hunk 11]
```diff
@@ -60,31 +60,33 @@ Note: This cmdlet does not support creating dynamic volumes.
...
 
-This line formats all newly partitioned disks.
-PS C:\>Format-Volume
+### Example 4: Create a Windows/system partition on MBR disk 0
+```powershell
+PS C:\> New-Partition -DiskNumber 0 -Size 100GB -MbrType IFS -IsActive
 ```
 
-This example uses five cmdlets and the pipeline to get all disks, filter them for only RAW, unpartitioned disks, initialize the disks, partition the disks, and then format them.
+This example creates a new Windows/system partition on MBR disk 0 with size 100GB.
```
Comment (JohanFreelancer9) [inline docset/windows/storage/New-Partition.md:89]:
```suggestion
This example creates a new Windows/system partition on MBR disk 0 with a size of 100 GB.
```

File: docset/winserver2012r2-ps/storage/New-Partition.md [hunk 12]
```diff
@@ -59,31 +59,33 @@ Note: This cmdlet does not support creating dynamic volumes.
...
+
+### Example 3: Create a new EFI partition on on GTP disk 2
+```powershell
+PS C:\> New-Partition -DiskNumber 2 -Size 500MB -GptType "{c12a7328-f81f-11d2-ba4b-00a0c93ec93b}"
+```
 
-This line initializes all Disk objects in the pipeline and then pipes the objects to the next cmdlet.
-PS C:\>Initialize-Disk -PassThru |
+This example creates a new EFI partition on disk 2 with size 500MB.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/storage/New-Partition.md:80]:
```suggestion
This example creates a new EFI partition on disk 2 with a size of 500 MB.
```

File: docset/winserver2012r2-ps/storage/New-Partition.md [hunk 13]
```diff
@@ -59,31 +59,33 @@ Note: This cmdlet does not support creating dynamic volumes.
...
 
-This line formats all newly partitioned disks.
-PS C:\>Format-Volume
+### Example 4: Create a Windows/system partition on MBR disk 0
+```powershell
+PS C:\> New-Partition -DiskNumber 0 -Size 100GB -MbrType IFS -IsActive
 ```
 
-This example uses five cmdlets and the pipeline to get all disks, filter them for only RAW, unpartitioned disks, initialize the disks, partition the disks, and then format them.
+This example creates a new Windows/system partition on MBR disk 0 with size 100GB.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/storage/New-Partition.md:88]:
```suggestion
This example creates a new Windows/system partition on MBR disk 0 with a size of 100 GB.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: docset/winserver2012r2-ps/storage/New-Partition.md [hunk 14]
```diff
@@ -59,31 +59,33 @@ Note: This cmdlet does not support creating dynamic volumes.
...
+PS C:\> Get-Disk | Where-Object PartitionStyle -Eq "RAW" | Initialize-Disk -PassThru | New-Partition -AssignDriveLetter -UseMaximumSize | Format-Volume
 ```
-This line gets all disk objects and then pipes the objects to the next command.
-PS C:\>Get-Disk |
 
-This line selects only objects where the PartitionStyle property value equals "RAW", and then pipes the objects to the next command.
-PS C:\>Where-Object PartitionStyle -Eq "RAW" |
+This example uses five cmdlets and the pipeline to get all disks, filter them for only RAW, unpartitioned disks, initialize the disks, partition the disks, and then format them.
+
+### Example 3: Create a new EFI partition on on GTP disk 2
```
Comment (velkovb) [inline docset/winserver2012r2-ps/storage/New-Partition.md:75]:
```suggestion
### Example 3: Create a new EFI partition on GTP disk 2
```

File: docset/windows/storage/New-Partition.md [hunk 15]
```diff
@@ -60,31 +60,33 @@ Note: This cmdlet does not support creating dynamic volumes.
...
+This example uses five cmdlets and the pipeline to get all disks, filter them for only RAW, unpartitioned disks, initialize the disks, partition the disks, and then format them.
+
+### Example 3: Create a new EFI partition on on GTP disk 2
+```powershell
+PS C:\> New-Partition -DiskNumber 2 -Size 500MB -GptType "{c12a7328-f81f-11d2-ba4b-00a0c93ec93b}"
+```
 
-This line initializes all Disk objects in the pipeline and then pipes the objects to the next cmdlet.
-PS C:\>Initialize-Disk -PassThru |
+This example creates a new EFI partition on disk 2 with size 500MB.
```
Comment (velkovb) [inline docset/windows/storage/New-Partition.md:81]:
```suggestion
This example creates a new EFI partition on disk 2 with size 500 MB.
```

File: docset/windows/storage/New-Partition.md [hunk 16]
```diff
@@ -60,31 +60,33 @@ Note: This cmdlet does not support creating dynamic volumes.
...
+This example uses five cmdlets and the pipeline to get all disks, filter them for only RAW, unpartitioned disks, initialize the disks, partition the disks, and then to format them.
+
+### Example 3: Create a new EFI partition on GTP disk 2
+```powershell
+PS C:\> New-Partition -DiskNumber 2 -Size 500MB -GptType "{c12a7328-f81f-11d2-ba4b-00a0c93ec93b}"
+```
 
-This line initializes all Disk objects in the pipeline and then pipes the objects to the next cmdlet.
-PS C:\>Initialize-Disk -PassThru |
+This example creates a new EFI partition on disk 2 with size 500 MB.
```
Comment (JohanFreelancer9) [inline docset/windows/storage/New-Partition.md:81]:
```suggestion
This example creates a new EFI partition on disk 2 with a size of 500 MB.
```

Comment (JohanFreelancer9) [reply docset/windows/storage/New-Partition.md:81]: Apologies @velkovb I seem to have missed this one.

Comment (velkovb) [reply docset/windows/storage/New-Partition.md:81]: Ah, thanks for spotting that. Seems like I did half of it :)

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Pull request merged**